test/pages/NavigationPage.ts

```diff
@@ -0,0 +1,43 @@
+import { Locator, Page } from "@playwright/test"
+import { BASE_URL, VALID_EMAIL, VALID_PASSWORD } from "../helper-config";
+import { LoginPage } from "./LoginPage";
+
+export class NavigationPage {
+    page : Page;
+    myBooksLink: Locator;
+    textToLogin: Locator;
+    loginPage: LoginPage;
+    booksDropdown: Locator;
+    browseLink: Locator;
+    dropdownOnBrowse: Locator;
+    optionOnBrowseDropdown: Locator;
+
+    constructor({page}: {page: Page}) {
+        this.page = page;
+        this.myBooksLink = page.getByRole('link', {name: 'My Books'});
+        this.textToLogin = page.getByText('Log in to use your free Open Library card to borrow digital books from the nonprofit Internet Archive', {exact: true});
```

Comment:
shorter lines are easier to review
```suggestion
        this.textToLogin = page.getByText(
          'Log in to use your free Open Library card to borrow digital books from the nonprofit Internet Archive',
          {exact: true}
        );
```